docs/sdk/client-side-sdks/react/react-gettingstarted.md

```diff
@@ -27,12 +27,12 @@ It’s best to initialize DevCycle in your root component (App.jsx or App.tsx),
 
 :::
 
-## Non-Blocking
-
 [//]: # (wizard-initialize-start)
 
+## Non-Blocking
+
 The withDevCycleProvider higher-order component (HOC) initializes the React SDK and wraps your root component. This provider may cause your app
-to flicker when it is first rendered, as all DevCycle variables will return their default values until the SDK is initialized. To avoid this, use the usIsDevCycleInitialized hook.
```

Comment:
feel like the deleted portion here is still relevant, otherwise we're basically just telling people "yeah our SDK makes your page flicker"

Reply:
ah true, i'll add it back. we just won't be able to link to that section without breaking the wizard, but it's probably better to have something mentioning it

Reply:
could you wrap the name in backticks?

Reply:
yep, fixed